### PR TITLE
Agregue 1 carpeta nueva

### PR DESCRIPTION
**Pull request by arobles98**
Agregue los archivos en una carpeta nueva :')

### CODE REVIEW

File: EjemploFiguras/Circle.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        return this.area;
+    }
+
+    @Override public String getColor() {
+        return "Color: " + color;
+    }
+
+    public String setColor(String colorAsignado){
+        this.color = colorAsignado;
+        return this.color;
```
Comment (ichi0915) [inline EjemploFiguras/Circle.java]: The setters do not have a return, change it to void and remove the return.

File: EjemploFiguras/Circle.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    public double setRadio(double radioAsignado){
+        this.radius = radioAsignado;
+        return this.radius;
```
Comment (ichi0915) [inline EjemploFiguras/Circle.java]: The setters do not have a return, change it to void and remove the return.

File: EjemploFiguras/Rectangulo.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    double width = 0.0;
+    
+    @Override double area(){
+        this.area = this.length * this.width;
+        return this.area;
+    }
+
+    public void setProperties(double largoAsignado, double anchoAsignado){
+        this.length = largoAsignado;
+        this.width = anchoAsignado;
```
Comment (ichi0915) [inline EjemploFiguras/Rectangulo.java:15]: The setters do not have a return, change it to void and remove the return.

File: EjemploFiguras/Rectangulo.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override public String getColor() {
+        return "Color: " + color;
+    }
+
+    public String setColor(String colorAsignado){
+        this.color = colorAsignado;
+        return this.color;
```
Comment (ichi0915) [inline EjemploFiguras/Rectangulo.java]: The setters do not have a return, change it to void and remove the return.

File: EjemploFiguras/Triangulo.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        this.height= alturaAsignado;
+    }
+
+    @Override public String getColor() {
+        return "Color: " + color;
+    }
+
+    public String setColor(String colorAsignado){
+        this.color = colorAsignado;
+        return this.color;
```
Comment (ichi0915) [inline EjemploFiguras/Triangulo.java]: The setters do not have a return, change it to void and remove the return.

**Review by ichi0915 — COMMENTED** [review]
Esta perfecto, solo que las funciones de set no deben de regresar nada

File: EjemploFiguras/Circle.java (hunk 1 above)
Comment (arobles98) [reply EjemploFiguras/Circle.java]: Ohhhh true, let me fix that :)

**Pull request merged**